framework/include/task_manager/task_manager.h

```diff
@@ -300,7 +314,15 @@ void task_manager_clean_info(task_info_t **info);
  * @since TizenRT v2.0 PRE
  */
 void task_manager_clean_infolist(task_info_list_t **info_list);
-
+/**
+ * @cond
+ * @internal
+ * @brief function for run exit callback when task is terminated.
+ */
+int task_manager_run_exit_cb(int pid);
```

Comment:
User does not need to use this API. This is for Task Manager internal operation. Let's move an another header.
For example, 
rename task_manager_drv.h to task_manager_internal.h and move it there.